### DISCUSSION TITLE
Bidirectional text

### DISCUSSION THREAD

**Discussion by avidseeker** [mdn/mdn-community#711] [Content] [answered]
MDN does not include much documentation about Bidirectional Text. Here are some resources:

* [W3C Bidi Text Basics](https://www.w3.org/International/articles/inline-bidi-markup/uba-basics)
* [RTL styling guide by Ahmed Shadeed](https://rtlstyling.com/posts/rtl-styling)
* [Freedesktop Bidi Spec](https://www.freedesktop.org/wiki/Specifications/bidi-spec/)

**Comment by bsmth** [comment, marked as answer]
Thanks for opening this one. We have a few pages that cover writing modes like:

* https://developer.mozilla.org/en-US/docs/Web/CSS/direction
* https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode
* https://developer.mozilla.org/en-US/docs/Web/CSS/unicode-bidi
* https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir

But maybe we're missing a good overview / guide page. I'll transfer this over as a content issue for now.
